src/Form/PublisherFormType.php

```diff
@@ -4,19 +4,40 @@
 
 use App\Entity\Publisher;
 use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
 use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\Length;
+use Symfony\Component\Validator\Constraints\NotBlank;
 
 class PublisherFormType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('name')
-            ->add('value')
-            ->add('country')
-            ->add('year')
+            ->add('name', TextType::class, [
+                'constraints' => [
+                    new NotBlank()
+                ]
+            ])
+            ->add('value', TextType::class, [
```

Comment:
jeigu cia pinigais value, tai pasicheckink sita:
https://symfony.com/doc/current/reference/forms/types/money.html
galimai ir entity reiks paeditinti, nesu tikras db :)
negerai palikti tokius fieldus text type, nes cia galiu belenka submitinti, turetu formos strictly buti tai info, kuri ir turi buti, be galimybes ivesti belenka useriui.